frontend/js/tests/recommended/Recommendations.test.tsx

```diff
@@ -1,406 +1,234 @@
-      const wrapper = mount<Recommendations>(
-        <GlobalAppContext.Provider value={mountOptions.context}>
-          <BrowserRouter>
-            <Recommendations {...props} />
-          </BrowserRouter>
-        </GlobalAppContext.Provider>
-      );
-      const instance = wrapper
-        .find(Recommendations)
-        .instance() as Recommendations;
-      const afterRecommendationsDisplaySpy = jest.spyOn(
-        instance,
-        "afterRecommendationsDisplay"
-      );
-      await act(async () => {
-        instance.setState({
-          currRecPage: 3,
-          recommendations: props.recommendations.slice(50, 73),
-        });
-      });
-
-      await instance.handleClickPrevious();
-      await waitForComponentToPaint(wrapper);
-
-      expect(instance.state.loading).toBeFalsy();
-      expect(instance.state.currRecPage).toEqual(2);
-      expect(instance.state.totalRecPages).toEqual(3);
-      expect(instance.state.recommendations).toEqual(
-        props.recommendations.slice(25, 50)
-      );
-      expect(afterRecommendationsDisplaySpy).toHaveBeenCalledTimes(1);
+  it("does not call loadFeedback if user is not the currentUser", async () => {
+    renderWithProviders(
+      <RouterProvider router={router} />,
+      {
+        currentUser: { name: "tarzan" },
+      },
+      {wrapper: reactQueryWrapper},
+      false
+    );
+    await waitFor(() => {
+      // Wait for data to be successfully loaded
+      const state = queryClient.getQueryState(queryKey);
+      expect(state?.status === "success").toBeTruthy();
     });
+    // Ensure feedback elements are not present
+    expect(recAPIFeedbackSpy).not.toHaveBeenCalled();
   });
 
-  describe("handleClickNext", () => {
-    it("doesn't do anything if already on last page", async () => {
-      const wrapper = mount<Recommendations>(
-        <GlobalAppContext.Provider value={mountOptions.context}>
-          <BrowserRouter>
-            <Recommendations {...props} />
-          </BrowserRouter>
-        </GlobalAppContext.Provider>
-      );
-      const instance = wrapper
-        .find(Recommendations)
-        .instance() as Recommendations;
-      const afterRecommendationsDisplaySpy = jest.spyOn(
-        instance,
-        "afterRecommendationsDisplay"
-      );
-      await act(async () => {
-        instance.setState({
-          currRecPage: 3,
-          recommendations: props.recommendations.slice(50, 74),
-        });
-      });
-
-      await instance.handleClickNext();
-      await waitForComponentToPaint(wrapper);
-
-      expect(instance.state.loading).toBeFalsy();
-      expect(instance.state.currRecPage).toEqual(3);
-      expect(instance.state.totalRecPages).toEqual(3);
-      expect(instance.state.recommendations).toEqual(
-        props.recommendations.slice(50, 73)
-      );
-      expect(afterRecommendationsDisplaySpy).toHaveBeenCalledTimes(0);
+  /* it("updates the recommendation feedback when clicked", async () => {
```

Comment:
Just wanted to point out this test is commented out, in case it is not on purpose.